backend/danswer/dynamic_configs/store.py

```diff
@@ -64,8 +27,9 @@ def get_session(self) -> Iterator[Session]:
             session.close()
 
     def store(self, key: str, val: JSON_ro, encrypt: bool = False) -> None:
-        # The actual encryption/decryption is done in Postgres, we just need to choose
-        # which field to set
+        # Not encrypted in Redis, but encrypted in Postgres
```

Comment:
should be safe right?

Reply:
What are we looking for in terms of "safe"?  It's possible to encrypt the connection to redis and set a password for the redis instance so that's arguably pretty "safe".  None of this is currently persisted to disk so that's also "safe".  Are there other considerations for us to look at?

Reply:
Seems good enough, just wanted to double check